rust/src/rfb/logger.rs

```diff
@@ -67,15 +69,14 @@ fn log_rfb(tx: &RFBTransaction, js: &mut JsonBuilder) -> Result<(), JsonError> {
         _ => (),
     }
     if let Some(security_result) = &tx.tc_security_result {
-        let _ = match security_result.status {
-            0 => js.set_string("security_result", "OK")?,
-            1 => js.set_string("security-result", "FAIL")?,
```

Comment:
Just noticed, this one would have been different (`-` instead of `_`) so this actually fixed a potential bug!

Reply:
Is this a bug in 7.0.x?

Reply:
Rather an inconsistency, but yes. The current code in 7.0.x uses a slightly different field name for `FAIL`.

Reply:
Nice accidental fix :-)

https://redmine.openinfosecfoundation.org/issues/7198 for the bug in 7

Reply:
I can provide a patch for 7.0.7

Reply:
https://github.com/OISF/suricata/pull/11606 if you want

Reply:
@catenacyber @satta From the comments here, was the field name changed in master as well? If so, I think we need an upgrade note.

Reply:
The field name was inconsistent depending on the value, most times with underscore, once with a dash, this last case was fixed